README.md

```diff
@@ -41,6 +41,7 @@ Run the following on your project directory to build the project and push to a d
 * `--docker-context-dir` Docker build context. Optional, to be used alongside `--dockerfile`.
 * `--docker-compose-file` Docker Compose file to be invoked for build and push.
 * `--docker-build-arg` Build arguments to be passed to docker build or docker-compose build. This parameter can be specified multiple times.
+* `--docker-secret-build-arg` Build arguments to be passed to docker build or docker-compose build. The argument value contains secret which will be hidden from the log. This parameter can be specified multiple times.
```

Comment:
Should we put some sort of disclaimer here? Passing secrets through command line isn't really safe anyways since command history, etc. will still be kept on the machine. It's fine in our particular case but for the generic builder it's a bit iffy. I.e. similar to the warning message you see from Docker. This also won't protect people from doing things like `echo $var` in `docker logs` but so it might be a tad misleading.

Reply:
I can think of reading the value from a file, similar to how docker saves the credential in the config file. If we choose that route, I'd like to do it for all other credentials we pass through the command line.